react/src/components/ServiceLauncherModal.tsx

```diff
@@ -636,11 +763,64 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
                       required
                     >
                       <Suspense fallback={<Skeleton.Input active />}>
-                        <VFolderLazyView uuid={endpoint?.model} />
+                        <VFolderLazyView
+                          uuid={endpoint?.model}
+                          mountDestination={endpoint?.model_mount_destination}
+                        />
                       </Suspense>
                     </Form.Item>
                   )
                 )}
+                <Flex
+                  direction="row"
+                  gap={'xxs'}
+                  align="stretch"
+                  justify="between"
+                >
+                  <Form.Item
+                    name={'modelMountDestination'}
+                    label={t('modelService.ModelMountDestination')}
+                    style={{ width: '50%' }}
+                    labelCol={{ style: { width: 400 } }}
+                  >
+                    <Input
+                      allowClear
+                      placeholder={'/models'}
+                      disabled={endpoint ? true : false}
+                    />
+                  </Form.Item>
+                  <MinusOutlined
+                    style={{ fontSize: token.fontSizeXL }}
+                    rotate={290}
+                  />
+                  <Form.Item
```

Comment:
If this is only for the dividing, you can use `<Divider type="vertical"/>`. [ref](https://ant.design/components/divider#divider-demo-vertical)

Reply:
I just added as if looks like a path divider, (`/`)